SmartcarAuth/VehicleInfo.swift

```diff
@@ -20,16 +20,28 @@
  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-@objc public class VehicleInfo: NSObject {
+/**
+ VehicleInfo class is used to describe the vehicle information that may be returned via query parameters in the case of authentication failure.
+ */
+@objcMembers public class VehicleInfo: NSObject {
     var vin: String?
     var make: String?
     var model: String?
-    var year: Int?
+    var year: NSNumber?
```

Comment:
just curious, why use `NSNumber` over `Int`?

Reply:
Int is not a supported optional type in Objective-C, so to make this class portable we need to use NSNumber!